src/geometry/euclidean.lean

```diff
@@ -5,8 +5,10 @@ Author: Joseph Myers.
 -/
 import analysis.normed_space.real_inner_product
 import analysis.normed_space.add_torsor
+import tactic.apply_fun
 
 noncomputable theory
+open_locale classical
```

Comment:
There should be notation for `pi` available. I don't know how it's set up at the moment. Maybe `open_locale real` will do the job. If it doesn't we should fix that.

Reply:
`localized "notation `π` := real.pi" in real` goes right after definition of `real.pi`.

Reply:
Now using π everywhere.